### PR TITLE
Senior copy updates (missing one string in Spanish translation)

### PR DESCRIPTION
**Pull request by nihonjinrxs**
Updates CSC address copy for Senior form content in SimpliGov in all languages.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207767919269297

### CODE REVIEW

**Review by lukecamelo — APPROVED** [review]
pending the missing translation, looks good!

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 1]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>For hours and more information about the <a href="https://www.mbta.com/fares/charliecard-store" target="_blank">CharlieCard Store <i class="fa fa-external-link" aria-hidden="true"></i></a>, visit the MBTA website.</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">The CharlieCard Store is now the Charlie Service Center and has been permanently relocated.</div>
+            <p>The Charlie Service Center (formerly the CharlieCard Store) is now located at 294 Washington St, Boston MA, 02108.</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:426]:
```suggestion
            <p>The Charlie Service Center (formerly the CharlieCard Store) is now located at 294 Washington St, Boston, MA 02108.</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 2]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>Para saber o horário de funcionamento e outras informações sobre a <a href="https://www.mbta.com/fares/charliecard-store" target="_blank">CharlieCard Store <i class="fa fa-external-link" aria-hidden="true"></i></a>, visite o site da MBTA.</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">A CharlieCard Store passa agora a ser chamada de Charlie Service Center e foi permanentemente realocada.</div>
+            <p>O Charlie Service Center (anteriormente CharlieCard Store) está agora localizado em 294 Washington St., Boston MA, 02108.</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:456]:
```suggestion
            <p>O Charlie Service Center (anteriormente CharlieCard Store) está agora localizado em 294 Washington St., Boston, MA 02108.</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 3]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>关于<a href="https://www.mbta.com/fares/charliecard-store" target="_blank">查理卡出售店 <i class="fa fa-external-link" aria-hidden="true"></i></a>的营业时间和更多信息，请访问MBTA网站。</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">查理卡出售店现在是查理服务中心，已永久搬迁。</div>
+            <p>查理服务中心（原查理卡出售店）现位于294 Washington St, Boston MA, 02108。</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:471]:
```suggestion
            <p>查理服务中心（原查理卡出售店）现位于294 Washington St, Boston, MA 02108。</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 4]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>關於<a href="https://www.mbta.com/fares/charliecard-store" target="_blank">查理卡出售店 <i class="fa fa-external-link" aria-hidden="true"></i></a>的營業時間和更多資訊，請查詢MBTA網站。</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">查理卡出售店現在是查理服務中心，已永久搬遷。</div>
+            <p>查理服務中心（原查理卡出售店）現位於294 Washington St, Boston MA, 02108。</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:486]:
```suggestion
            <p>查理服務中心（原查理卡出售店）現位於294 Washington St, Boston, MA 02108。</p>
```

File: assets/senior_html/form_elements/generated-content.txt [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  WHEN (@FORMelement3 = 'English') THEN 'Your card is currently available for pick up at the Charlie Service Center (formerly the CharlieCard Store) at its new location:'
+  WHEN (@FORMelement3 = 'Español') THEN 'Su tarjeta ya está disponible para recogerla en el Centro de Servicio Charlie (anteriormente Tienda CharlieCard) en su nueva ubicación:'
+  WHEN (@FORMelement3 = 'Português') THEN 'No momento, o cartão está disponível para retirada no Charlie Service Center (antigo CharlieCard Store) no novo local:'
+  WHEN (@FORMelement3 = '中文简体') THEN '你现在可以在查理服务中心（原查理卡出售店）的新地址领取你的卡：'
+  WHEN (@FORMelement3 = '中文繁體') THEN '你現在可以在查理服務中心（原查理卡出售店）的新地址領取你的卡：'
+END
+
+<!-- GENERATED CONTENT | CHARLIECARD ADDRESS LINE 2 | FORM ELEMENT 197 -->
+CASE
+  WHEN Not IsDefault(@FORMelement3) THEN '294 Washington St'
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/generated-content.txt:12]:
```suggestion
  WHEN Not IsDefault(@FORMelement3) THEN '296 Washington St'
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 6]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>For hours and more information about the <a href="https://www.mbta.com/fares/charliecard-store" target="_blank">CharlieCard Store <i class="fa fa-external-link" aria-hidden="true"></i></a>, visit the MBTA website.</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">The CharlieCard Store is now the Charlie Service Center and has been permanently relocated.</div>
+            <p>The Charlie Service Center (formerly the CharlieCard Store) is now located at 294 Washington St, Boston, MA 02108.</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:426]:
```suggestion
            <p>The Charlie Service Center (formerly the CharlieCard Store) is now located at 296 Washington St, Boston, MA 02108.</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 7]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>Para conocer los horarios y más información sobre la <a href="https://www.mbta.com/fares/charliecard-store" target="_blank">Tienda CharlieCard <i class="fa fa-external-link" aria-hidden="true"></i></a>, visite la página web de la MBTA.</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">The CharlieCard Store is now the Charlie Service Center and has been permanently relocated.</div>
+            <p>El Centro de Servicio Charlie (anteriormente Tienda CharlieCard) ahora se encuentra ubicado en 294 Washington St, Boston, MA 02108.</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:441]:
```suggestion
            <p>El Centro de Servicio Charlie (anteriormente Tienda CharlieCard) ahora se encuentra ubicado en 296 Washington St, Boston, MA 02108.</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 8]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>Para saber o horário de funcionamento e outras informações sobre a <a href="https://www.mbta.com/fares/charliecard-store" target="_blank">CharlieCard Store <i class="fa fa-external-link" aria-hidden="true"></i></a>, visite o site da MBTA.</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">A CharlieCard Store passa agora a ser chamada de Charlie Service Center e foi permanentemente realocada.</div>
+            <p>O Charlie Service Center (anteriormente CharlieCard Store) está agora localizado em 294 Washington St., Boston, MA 02108.</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:456]:
```suggestion
            <p>O Charlie Service Center (anteriormente CharlieCard Store) está agora localizado em 296 Washington St., Boston, MA 02108.</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 9]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>关于<a href="https://www.mbta.com/fares/charliecard-store" target="_blank">查理卡出售店 <i class="fa fa-external-link" aria-hidden="true"></i></a>的营业时间和更多信息，请访问MBTA网站。</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">查理卡出售店现在是查理服务中心，已永久搬迁。</div>
+            <p>查理服务中心（原查理卡出售店）现位于294 Washington St, Boston, MA 02108。</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:471]:
```suggestion
            <p>查理服务中心（原查理卡出售店）现位于296 Washington St, Boston, MA 02108。</p>
```

File: assets/senior_html/form_elements/senior-form-fields.html [hunk 10]
```diff
@@ -416,29 +416,79 @@ <h1>得到你的卡</h1>
...
-
-<p>關於<a href="https://www.mbta.com/fares/charliecard-store" target="_blank">查理卡出售店 <i class="fa fa-external-link" aria-hidden="true"></i></a>的營業時間和更多資訊，請查詢MBTA網站。</p>
+<div class="info-alert-item">
+    <div class="info-alert-item__icon" aria-hidden="true">
+        <span class="notranslate fa fa-exclamation-circle fa-2x"></span>
+    </div>
+    <div class="info-alert-item__top">
+        <div role="alert" class="info-alert-item__top-text-container">
+            <div class="info-alert-item__effect">查理卡出售店現在是查理服務中心，已永久搬遷。</div>
+            <p>查理服務中心（原查理卡出售店）現位於294 Washington St, Boston, MA 02108。</p>
```
Comment (nihonjinrxs) [inline assets/senior_html/form_elements/senior-form-fields.html:486]:
```suggestion
            <p>查理服務中心（原查理卡出售店）現位於296 Washington St, Boston, MA 02108。</p>
```

**Pull request merged**